README.md

```diff
@@ -1,2 +1,45 @@
-# portfolio
+# Portfolio
 The portfolio of projects I developed
+
+## Built With
+
+- html
+- css
+- linters
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] Please, make sure there are no broken links in your README file because it's the first impression a visitor will take while looking at your repo. :computer: